### PR TITLE
fix(frontend): Resolve all warnings when running `npm run start`

### PR DESCRIPTION
**Pull request by zijianjoy**
**Description of your changes:**

Make sure to resolve any styling and ts usage warnings during development.

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

**Review by zpChris — APPROVED** [review]
LGTM, thank you for making these fixes @zijianjoy! ✅

File: frontend/src/components/NewRunParametersV2.tsx [hunk 1]
```diff
@@ -130,6 +130,7 @@ function generateInputValidationErrMsg(
 }
 
 function NewRunParametersV2(props: NewRunParametersProps) {
+  const { specParameters, setIsValidInput } = props;
```
Comment (jlyaoyuli) [inline frontend/src/components/NewRunParametersV2.tsx:133]: I don't fully get the change here. Can you describe a little more details? Thank you.

**Comment by jlyaoyuli** [comment]
/lgtm
/approve
Thank you @zijianjoy for fixing these warnings.

**Pull request merged**